lib/service/connect.go

```diff
@@ -447,18 +447,35 @@ func (process *TeleportProcess) periodicSyncRotationState() error {
 	}
 
 	periodic := interval.New(interval.Config{
-		Duration:      defaults.HighResPollingPeriod,
-		FirstDuration: utils.HalfJitter(defaults.HighResPollingPeriod),
+		Duration:      pollingInterval,
+		FirstDuration: utils.HalfJitter(pollingInterval),
 		Jitter:        utils.NewSeventhJitter(),
 	})
 	defer periodic.Stop()
 
+	errors := utils.NewTimedCounter(process.Clock, syncRotationFailureThreshold.Time)
+
 	for {
 		err := process.syncRotationStateCycle()
 		if err == nil {
 			return nil
 		}
-		process.log.Warningf("Sync rotation state cycle failed: %v, going to retry after ~%v.", err, defaults.HighResPollingPeriod)
+		process.log.WithError(err).Warningf("Sync rotation state cycle failed: %v", err)
```

Comment:
Avoid logging err twice.

```suggestion
		process.log.WithError(err).Warning("Sync rotation state cycle failed")
```